packages/fuel-indexer/src/lib.rs

```diff
@@ -83,4 +83,6 @@ pub enum IndexerError {
     NativeExecutionRuntimeError,
     #[error("Tokio time error: {0:?}")]
     Elapsed(#[from] tokio::time::error::Elapsed),
+    #[error("{0}.")]
+    OtherError(String),
```

Comment:
Nit
```suggestion
    Unknown(String),
```
(To be consistent with how we do thing elsewhere)